### DISCUSSION TITLE
GCP Memstore Discovery End Point connectivity

### DISCUSSION THREAD

**Discussion by hk250097** [redis/lettuce#2902] [General]
Hello,

We are using spring session which uses Lettuce connection factory to connect to the GCP redis single instance.
We are planning to move to GCP redis cluster which has Discovery End point (single IP backed by multiple redis nodes)
How to integrate lettuce connectivity with spring session.
Is there any specific spring boot version which supports GCP memstore Discovery End Point connectivity..

**Comment by tishun** [comment]
Does [this](https://cloud.google.com/memorystore/docs/cluster/client-library-connection) page help answer the question?
